components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IEngineConfiguration.cs

```diff
@@ -37,5 +37,10 @@ public interface IEngineConfiguration {
         /// Default: 4096 messages with 256KB ends up in 1 GB memory consumed.
         /// </summary>
         int? MaxOutgressMessages { get; }
+
+        /// <summary>
+        /// Flag to determine if a telemetry routing info is required.
+        /// </summary>
+        bool? HasRoutingInfo { get; }
```

Comment:
Since it is a enable/disable flag, shall we call it `EnableRoutingInfo` instead ?

Reply:
done